sources/sections-a1-en/18-16th-cgpm.adoc

```diff
@@ -64,13 +64,13 @@ The 16th Conférence Générale des Poids et Mesures (CGPM),
 
 *recognizing* the general principles adopted for writing the unit symbols in Resolution 7 of the 9th CGPM (1948),
 
-*considering* that the symbol stem:["unitsml(l)"] for the unit litre was adopted by the Comité International des Poids et Mesures (CIPM) in 1879 and confirmed in the same Resolution of 1948,
+*considering* that the symbol stem:[sf "l"] for the unit litre was adopted by the Comité International des Poids et Mesures (CIPM) in 1879 and confirmed in the same Resolution of 1948,
 
-*considering* also that, in order to avoid the risk of confusion between the letter l and the number 1, several countries have adopted the symbol stem:["unitsml(L)"] instead of stem:["unitsml(l)"] for the unit litre,
+*considering* also that, in order to avoid the risk of confusion between the letter l and the number 1, several countries have adopted the symbol stem:[sf "L"] instead of stem:[sf "l"] for the unit litre,
 
 *considering* that the name litre, although not included in the Système International d'Unités, must be admitted for general use with the System,
 
-*decides*, as an exception, to adopt the two symbols stem:["unitsml(l)"] and stem:["unitsml(L)"] as symbols to be used for the unit litre, footnote:[The CIPM, in 1990, considered that it was still too early to choose a single symbol for the litre.]
+*decides*, as an exception, to adopt the two symbols stem:[sf "l"] and stem:[sf "L"] as symbols to be used for the unit litre, footnote:[The CIPM, in 1990, considered that it was still too early to choose a single symbol for the litre.]
```

Comment:
Leave them as UnitsML. This is a UnitsML rendering issue.

Reply:
Done.